nullabilitynotes.md

```diff
@@ -0,0 +1,105 @@
+
```

Comment:
I think this file may have snuck in as part of your nullable references work.

Reply:
Ah, yes. I was doing so well having both branches going for a while there.